spec/features/admin_approve_delete_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+feature 'admin manages vendor page', %{
+  As an admin
+  I want to delete or approve a vendor
+  So that users can see appropriate vendors
+} do
+
+  let!(:category) { FactoryGirl.create(:category) }
+
+  scenario 'admin approves a valid vendor' do
+    member = FactoryGirl.create(:user, user_type: 'Admin')
```

Comment:
I would rename this variable `admin` - it makes it more explicit what kind of user you're talking about.